private/buf/cmd/buf/command/dep/internal/internal.go

```diff
@@ -92,8 +92,7 @@ func Prune(
 		switch t := malformedDep.Type(); t {
 		case bufworkspace.MalformedDepTypeUnused:
 			logger.Sugar().Warnf(
-				`Module %s is declared in your buf.yaml deps but is unused. This command only modifies buf.locks, not buf.yamls, please %s from your buf.yaml deps if it is not needed.`,
-				malformedDep.ModuleFullName(),
+				`Module %[1]s is declared in your buf.yaml deps but is unused. This command only modifies buf.lock files, not buf.yaml files. Please remove %[1]s from your buf.yaml deps if it is not needed.`,
```

Comment:
What is it updating it to? Can you provide an example?

Reply:
Before:
```
$ buf dep update
WARN	Module buf.build/protocolbuffers/wellknowntypes is declared in your buf.yaml deps but is unused. This command only modifies buf.locks, not buf.yamls, please buf.build/protocolbuffers/wellknowntypes from your buf.yaml deps if it is not needed.
```

After:
```
$ buf dep update
WARN	Module buf.build/protocolbuffers/wellknowntypes is declared in your buf.yaml deps but is unused. This command only modifies buf.lock files, not buf.yaml files. Please remove buf.build/protocolbuffers/wellknowntypes from your buf.yaml deps if it is not needed.
```

The main change is to add `remove` after `please`.